static/app/views/explore/savedQueries/savedQueriesTable.tsx

```diff
@@ -221,13 +221,13 @@ export function SavedQueriesTable({
                 <Tooltip title={'Sentry'}>
                   <ActivityAvatar type="system" size={20} />
                 </Tooltip>
-              ) : (
-                <Avatar
-                  user={query.createdBy}
-                  tooltip={query.createdBy?.name}
+              ) : query.createdBy ? (
+                <ActorAvatar
+                  actor={query.createdBy}
+                  tooltip={query.createdBy.name}
                   hasTooltip
                 />
-              )}
+              ) : null}
```

Comment:
It was passing undefined to the `UserAvatar` before, which would just render `null`. See https://github.com/getsentry/sentry/blob/34372aaeb000a6a95601e80dafc088c0c621eee6/static/app/components/core/avatar/userAvatar.tsx#L28-L31